doc/monitoring/metrics_reference.rst

```diff
@@ -207,3 +207,87 @@ Runtime
 * ``tnt_runtime_lua`` – Lua garbage collector size in bytes
 
 * ``tnt_runtime_used`` - number of bytes used for Lua runtime
+
+.. _metrics-cartridge:
+
+-------------------------------------------------------------------------------
+Cartridge
+-------------------------------------------------------------------------------
+
+* ``cartridge_issues`` - Number of
+  `issues <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge.issues/>`
+  across cluster instances. This metric always has labels - ``{level="critical"}``.
+  ``level`` - the level of the issue. ``critical`` level is associated with critical
+  cluster problems, e.g. memory used ratio > 90%, ``warning`` is associated with
+  other cluster problems, e.g. replication issues on cluster.
+
+.. _metrics-luajit:
+
+-------------------------------------------------------------------------------
+LuaJIT metrics
+-------------------------------------------------------------------------------
+
+LuaJIT metrics help to understand Lua GC state. Only in Tarantool 2.6+.
+
+**General JIT metrics:**
+
+* ``lj_jit_snap_restore`` - overall number of snap restores
+
+* ``lj_jit_trace_num`` - amount of JIT traces
+
+* ``lj_jit_trace_abort`` - overall number of abort traces
+
+* ``lj_jit_mcode_size`` - total size of all allocated machine code areas
+
+**JIT strings:**
+
+* ``lj_strhash_hit`` - number of strings being interned
+
+* ``lj_strhash_miss`` - total number of string allocations
+
+**GC steps:**
+
+* ``lj_gc_steps_atomic`` - count of incremental GC steps (atomic state)
+
+* ``lj_gc_steps_sweepstring`` - count of incremental GC steps (sweepstring state)
+
+* ``lj_gc_steps_finalize`` - count of incremental GC steps (finalize state)
+
+* ``lj_gc_steps_sweep`` - count of incremental GC steps (sweep state)
+
+* ``lj_gc_steps_propagate`` - count of incremental GC steps (propagate state)
+
+* ``lj_gc_steps_pause`` - count of incremental GC steps (pause state)
+
+**GC allocations:**
```

Comment:
I see that it's literally written `lj_gc_*` here, but in fact it's user-allocated objects, not the objects allocated by garbage collector. Am I missing something?